src/Juvix/Core/FromFrontend.hs

```diff
@@ -41,8 +41,7 @@ paramConstant k = do
     Nothing -> throwFF $ UnsupportedConstant k
```

Comment:
This entire file is going to change, so you might have to merge and deal with conflicts wtih the sexp changes that got merged